www/src/css/snaplist.css

```diff
@@ -34,6 +44,10 @@
   margin-top:140%;
 }
 
+.b-store .b-snaplist_grid .b-snaplist__item:before {
```

Comment:
OK, I get it. In situations like these simplifying CSS leads to complicating on the views part, so I guess we have to stick to a compromise. Thanks for clarification.